common-prod/common.tfvars

```diff
@@ -196,7 +196,7 @@ user_access_cidr_blocks = [
   "195.92.40.49/32",   # OMNI
   "62.232.198.64/28",  # I2N 
   "81.187.190.127/32", # Lazzurs Home
-  "3.10.56.113/32",    # PSN Proxy A
+  "3.10.115.64/32",    # PSN Proxy A
```

Comment:
are the user_access_cidr blocks used for psn proxy -> non spg things?

would make sense to have old + new both allowed for now, then delete old later